aws_lambda_powertools/utilities/parser/models/ses.py

```diff
@@ -1,12 +1,21 @@
+import logging
+import sys
 from datetime import datetime
-from typing import List, Optional
+from typing import List, NewType, Optional
 
 from pydantic import BaseModel, Field
-from pydantic.networks import EmailStr
 from pydantic.types import PositiveInt
 
 from ..types import Literal
 
+has_email_validator = "email_validator" in sys.modules
+
+if has_email_validator:
```

Comment:
we're gonna need `#pragma: nocover` too because this will decrease our tests despite being intended behaviour ;-)